src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -55,7 +55,7 @@ public function __construct(SecurityServiceInterface $securityService, HomePage
     }
 
     /**
-     * @Given I am logged in as :email
+     * @Given /^I am logged in as "([^""]*)"$/
```

Comment:
Because it catches also the `I am logged in as administrator` step, but we need only emails there :)